### ISSUE TITLE
For the test of custom data

### ISSUE THREAD

**Issue by cyy-1234** [tgxs002/HPSv2#9]
For the test of custom data, is the prompt word only contains the content, can not appear about the style of the word? And I want to know what outputs["image_features"], outputs["text_features"], outputs["logit_scale"] mean in score.py, And finally what it means to use hps_score[0] as a score,
![image](https://github.com/tgxs002/HPSv2/assets/59501159/38ff0664-3c85-4c8b-9078-05ad6e6c26e7)

**Comment by tgxs002** [comment]
The prompt can contain "style words", the model is capable of identifying these words, but our benchmark avoids using prompts with them to avoid potential bias. 
outputs["image_features"], outputs["text_features"] are normalized features of an image and a prompt respectively, and our model is trained to give higher similarity for images that are better preferred by human evaluators. outputs["logit_scale"] is no longer there in our code, you can try pulling our latest commits. 
hps_score[0] can be used to compare images generated with the same prompt, but not for comparing images with different prompts.

**Comment by tgxs002** [comment]
You can refer to our paper and [our repo](https://github.com/tgxs002/HPSv2#human-preference-dataset-v2) for detailed explanation.

**Comment by cyy-1234** [comment]
![image](https://github.com/tgxs002/HPSv2/assets/59501159/90146910-9d59-4243-be56-12ee698a51ac)
May I ask if the latest best score is the checkpoint used on July 26?

**Comment by cyy-1234** [comment]
Hello dear author, think about it, you have a classification of training data, such as people, animals, cars, food and other categories.

**Comment by w-zhih** [comment]
The "compressed checkpoint" is the checkpoint of preference prediction model "HPS v2". Compared with the last checkpoint, it just excludes some unnecessary data.

> ![image](https://user-images.githubusercontent.com/59501159/257155195-90146910-9d59-4243-be56-12ee698a51ac.png) May I ask if the latest best score is the checkpoint used on July 26?
